front/app/containers/Admin/messaging/texting/components/SMSCampaignForm.tsx

```diff
@@ -121,6 +125,10 @@ const SMSCampaignForm = ({
     }
   };
 
+  // no campaign = creating new SMS, campaign = updating existing draft
+  const buttonCopy = isNilOrError(campaignId)
+    ? 'Preview SMS'
+    : 'Update and preview SMS';
```

Comment:
The button is not displayed unless we're in draft mode. So either this won't have effect or we need to change the button show/hide condition.

Reply:
Edit: this shows in the new SMS form of course. Pls ignore the above comment. :)